src/redux/books/books.js

```diff
@@ -15,7 +15,7 @@ const initialState = [{
 export default function booksReducer(state = initialState, action = {}) {
   switch (action.type) {
     case ADD_BOOK:
-      return { ...state, book: action.payload };
+      return [...state, action.payload];
     case REMOVE_BOOK:
       return [...state.filter((book) => book.id !== action.payload.id)];
```

Comment:
- [ ] (OPTIONAL) Great job so far 👍 I can now add a book but I cannot remove a book. Kindly fix this issue so as to have the correct functionality of the app 💯